package Message; 
message WorldDataResponse							//同步游戏世界初始数据
{ 
		optional int64     playerid	= 1;  //玩家ID
   	optional int64     itemid		= 2;  //物品ID
   	optional int64     heroid		= 3;  //英雄ID
} 